### ISSUE TITLE
Expand groupBy functionality

### ISSUE THREAD

**Issue by devinivy** [balderdashy/waterline-sequel#47]
Currently groupBy requires an aggregate query, and the query returns _only_ the aggregate value.  I'd like to see groupBy not require an aggregate query, and return all meaningful rows instead of the one aggregate value.  This may be acheived using the `DISTINCT` keyword.  See original issue here with comments from @mikermcneil: https://github.com/balderdashy/waterline-adapter-tests/issues/13

This may involve nailing down some tests and specs from waterline core.  We also may need to make this a new interface or feature, @dmarcelino .

**Comment by dmarcelino** [comment]
:+1:

**Comment by sailsbot** [comment]
Thanks for posting, @devinivy.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
